src/cpu.rs

```diff
@@ -205,6 +206,20 @@ impl Speed for CPU {
             },
         }
     }
+
+    fn to_csv(&self) -> String {
+        format!(
+            "{},{},{},{},{},{},{},{}\n",
+            self.name,
+            self.number,
+            self.max_freq,
+            self.min_freq,
+            self.cur_freq,
+            self.cur_temp,
+            self.cur_usage,
+            self.gov
```

Comment:
Unix timestamp would be nice.

Reply:
Keep in mind that the rate of CVS logging would depend on the current delay for the daemon (I think). This makes plotting consistent data hard, if we add a timestamp to the logs it would make it much easier.

Reply:
oh true

Reply:
Fixed